src/navigation/context.py

```diff
@@ -204,15 +203,31 @@ def convert(waypoint: GPSWaypoint) -> Waypoint:
     return Waypoint(fiducial_id=waypoint.id, tf_id=f"course{waypoint.id}", type=waypoint.type), SE3(position=odom)
 
 
+def convert_cartesian_to_gps(coordinate: np.ndarray) -> GPSWaypoint:
+    """
+    Converts a coordinate to a GPSWaypoint (used for sending data back to basestation)
+    """
+    # read required parameters, if they don't exist an error will be thrown
+    REF_LAT = rospy.get_param("gps_linearization/reference_point_latitude")
+    REF_LON = rospy.get_param("gps_linearization/reference_point_longitude")
+    lat, lon, _ = pymap3d.enu2geodetic(
+        e=coordinate[0], n=coordinate[1], u=0.0, lat0=REF_LAT, lon0=REF_LON, h0=0.0, deg=True
```

Comment:
Seems like the majority of these two functions could be made into a single helper

Reply:
I moved the REF_LAT and REF_LON definitions to the top of the file where they should be and then its different since its two different functions so I think its ok now

Reply:
Yeah you're right it was only that on bit. Looks good